packages/application-config/schema.json

```diff
@@ -14,13 +14,6 @@
         "type": "string"
       },
       "uniqueItems": true
-    },
-    "hstsDirective": {
```

Comment:
Unrelated, but there is no need to keep this as a schema reference. We can simply inline it.